src/content/docs/opentelemetry/best-practices/opentelemetry-best-practices-resources.mdx

```diff
@@ -128,6 +129,26 @@ Relationships between a service and host entity require that the service
 includes the `host.id` resource attribute and that it matches the `host.id` of
 the host it is running on.
 
+### Service to AWS resources 
+
+#### Service to AWS Elasticache Memcached Cluster 
+
+Relationships between a service and AWS Memcached Cluster entity requires the following:
+
+Service should generate the following span attributes:
+
+* `db.system`: `memcached`
+* `net.peer.name`: AWS connection endpoint of the memcached clusterr ending with `cache.amazonaws.com`
+* `net.peer.port`: Port number on which memcached cluster is running eg: `11211`
```

Comment:
`net.peer.name` and `net.peer.port` are old attributes and they have been deprecated. There is probably still old instrumentation that produces them, but we're going to need to support the new attributes.

@jack-berg we should consider basing [this rule](https://github.com/newrelic/entity-definitions/blob/8dc1039797e90c227e49cf1a92278dd5faf80d07/relationships/synthesis/EXT-SERVICE-to-MEMCACHED.yml#L17-L24) based on the `apm.service.datastore.operation` metric we generate. That way our entity relationship definitions do not need to support evolving semantic conventions.

Reply:
See slack convo